Images Docs Update (Academy) (#361)
chainguard-dev/edu: content/chainguard/chainguard-images/faq.md

---

### Are Chainguard Images based on Alpine?

> No. Chainguard images are based on [Wolfi](/open-source/wolfi/), a Linux _undistro_ we built specifically to address software supply chain security issues.

Reviewer: What about:

Chainguard images are based on [Wolfi](/open-source/wolfi/), a Linux _undistro_ we built specifically to address software supply chain security issues. We do have some images with Alpine based variants in order to support musl or unusual architectures.

[Chainguard Images](https://www.chainguard.dev/chainguard-images?utm_source=docs) are officially maintained by [Chainguard](https://chainguard.dev) employees, but they are also open source, which means any community member is welcome to suggest improvements.

### Can I simply replace my current base image with a Chainguard Image and it will work out of the box?

> Most Chainguard Images are distroless, which means they are minimalist and most of them don't come with a package manager. Depending on your stack, you may need to include additional software by copying artifacts from multi-stage builds.

Reviewer: Most -> Many?

Reviewer 2: What about ...

```suggestion
Chainguard Images are designed/developed to be minimalist images. Because many are distroless, Chainguard Images don't usually come with a package manager. Depending on your stack, you may need to include additional software by copying artifacts from multi-stage builds.
```

Is the plan that the majority of them are distroless without a package manager?

Reviewer: I think the plan going forward will be to have versions with and without for most images. So e.g. "cgr.dev/chainguard/bla:dev" has a package manager but "cgr.dev/chainguard/bla:latest" doesn't. Not sure if the name will be dev/debug/full yet.
